woocommerce/woocommerce
# Add coming soon newsletter feature flag

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





Closes https://github.com/woocommerce/woocommerce/issues/52412

Create a feature flag for coming soon newsletter with default disabled

Similar to:

- https://github.com/woocommerce/woocommerce/pull/49966



### How to test the changes in this Pull Request:

Suggested scope of review: 
- Appropriateness of feature flag name
- That all required changes have been made for adding a feature flag

1. Should be no visible changes to WooCommerce core at this point
2. Should see the `coming-soon-newsletter` feature flag toggle present in WC Beta Tester plugin, which should do nothing when toggled



### Changelog entry







<details>

<summary>Changelog Entry Details</summary>

#### Significance




#### Type




#### Message 

</details>

<details>

<summary>Changelog Entry Comment</summary>

#### Comment 

</details>
